src/config/config.js

```diff
@@ -120,6 +120,14 @@ export default {
    * @property {Boolean} revocations_expire If true, expired revocation signatures are ignored
    */
   revocations_expire:       false,
+  /**
+   * @memberof module:config
+   * @property {Boolean} allow_insecure_decryption_with_signing_keys
+   *  If true, RSA sign-only keys can be used for decryption
```

Comment:
```suggestion
   *  If true, RSA signing keys can be used for decryption
```
This is very nitpicky, but otherwise people might think you're talking about the "RSA Sign-Only" algorithm (ID 3) as opposed to just RSA keys in general that have `keyFlags = sign`.
Also, even more nitpicky, but please change the double spaces at the beginning to single spaces and add some dots at the end :upside_down_face:

Reply:
But signing keys are not necessarily sign-only, you can have both sign + encrypt flags.

Reply:
I see. But I guess in the same vein, they could also be authentication subkeys, or have keyFlags = 0... so maybe it's best to just say "non-encryption keys" or "keys without encryption key flags"